microsoft/ReportingServicesTools
# Create Get-RsRestFolderContent.ps1

Initial commit of REST version of the Get-RsFolderContent function, for use in PS7+.

Fixes # 
None.

Changes proposed in this pull request:
New function.

How to test this code:
 - See Help examples.
 - `Get-RsRestFolderContent -ReportPortalUri 'http://localhost/reportserver_sql2012' -RsFolder /`
 - `Get-RsRestFolderContent -ReportServerUri http://localhost/ReportServer -RsFolder / -Recurse`

Has been tested on (remove any that don't apply):
 - Powershell 5.1 and above
 - Windows 10
 - Power BI Report Server
